raster/shortcuts.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import unicode_literals
+
+from raster.utils import colormap_to_rgba
+
+
+def set_session_colormap(session, key, colormap):
+    """ Store the colormap in the user session
```

Comment:
I usually add a linebreak after the first `"""` block and add a `.` at the end of the sentence in docstrings.